src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -11,6 +9,9 @@ class Fluke8846A(DMM):
     INSTR_TYPE = "VISA"
 
     def __init__(self, instrument, *args, **kwargs):
+        self.measurement_delay = (
+            0.2  # Delay between call to self.measurement() and querying the DMM.
```

Comment:
Can you pull the comment up to the line above to avoid the assignment being split across lines.